src/sass/partials/_general.scss

```diff
@@ -28,3 +28,12 @@ ol {
   padding: 0;
   margin: 0;
 }
+a {
+  color: #000000;
+  &:link {
+    text-decoration: none;
+  }
+  &hover {
```

Comment:
Brak dwukropka między &hover. Popraw na &:hover i zatwierdzam :). Możesz też to zapisać jeszcze krócej, żeby nie powtarzać "text-decoration: none;"

a {
  color: #000000;
  &:link,
  &:hover {
    text-decoration: none;
  }
}